synapse/cortex.py

```diff
@@ -86,7 +86,7 @@
 reqValidTagModel = s_config.getJsValidator({
     'type': 'object',
     'properties': {
-        # 'prune': {'type': 'boolean'},
+        'prune': {'type': 'number'},
```

Comment:
We should have a minimum range defined here https://json-schema.org/understanding-json-schema/reference/numeric.html#range